README.rst

```diff
@@ -135,71 +198,54 @@ And then, plug into a subscribable schema:
 
     schema = graphene.Schema(query=Query, subscription=Subscription)
 
-You can see a full example here:
-https://github.com/graphql-python/graphql-ws/tree/master/examples/flask_gevent
-
-Django Channels
-~~~~~~~~~~~~~~~
+Gevent compatible servers
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-First ``pip install channels`` and it to your django apps
-
-Then add the following to your settings.py
+Then just plug into your Gevent server, for example, Flask:
 
 .. code:: python
 
-    CHANNELS_WS_PROTOCOLS = ["graphql-ws", ]
-    CHANNEL_LAYERS = {
-        "default": {
-            "BACKEND": "asgiref.inmemory.ChannelLayer",
-            "ROUTING": "django_subscriptions.urls.channel_routing",
-        },
-
-    }
-
-Setup your graphql schema
+    from flask_sockets import Sockets
+    from graphql_ws.gevent import GeventSubscriptionServer
+    from schema import schema
 
-.. code:: python
-
-    import graphene
-    from rx import Observable
-
-
-    class Query(graphene.ObjectType):
-        hello = graphene.String()
+    subscription_server = GeventSubscriptionServer(schema)
+    app.app_protocol = lambda environ_path_info: 'graphql-ws'
 
-        def resolve_hello(self, info, **kwargs):
-            return 'world'
 
-    class Subscription(graphene.ObjectType):
+    @sockets.route('/subscriptions')
+    def echo_socket(ws):
+        subscription_server.handle(ws)
+        return []
 
-        count_seconds = graphene.Int(up_to=graphene.Int())
+You can see a full example here:
+https://github.com/graphql-python/graphql-ws/tree/master/examples/flask_gevent
 
+Django v1.x
+~~~~~~~~~~~
 
-        def resolve_count_seconds(
-            root,
-            info,
-            up_to=5
-        ):
-            return Observable.interval(1000)\
-                                .map(lambda i: "{0}".format(i))\
-                                .take_while(lambda i: int(i) <= up_to)
-
+For Django v1.x and Django Channels v1.x, setup your schema in ``settings.py``
 
+.. code:: python
 
-    schema = graphene.Schema(
-        query=Query,
-        subscription=Subscription
-    )
+    GRAPHENE = {
+        'SCHEMA': 'yourproject.schema.schema'
+    }
 
-Setup your schema in settings.py
+Then ``pip install "channels<1"`` and it to your django apps, adding the
+following to your ``settings.py``
 
 .. code:: python
 
-    GRAPHENE = {
-        'SCHEMA': 'path.to.schema'
+    CHANNELS_WS_PROTOCOLS = ["graphql-ws", ]
+    CHANNEL_LAYERS = {
+        "default": {
+            "BACKEND": "asgiref.inmemory.ChannelLayer",
+            "ROUTING": "django_subscriptions.urls.channel_routing",
+        },
```

Comment:
the example is out-dated. They retired this. I used:
```CHANNEL_LAYERS = {
    "default": {"BACKEND": "channels.layers.InMemoryChannelLayer",},
}
```
instead, and that works now

Reply:
This is in the outdated "old django" section anyway, is that backend configuration really retired in channels v1?

Reply:
From what I found `asgiref` retired their in-memory channel layer. So, if you get `asgiref`, it no longer has the class listed in the doc :-/

about the `ROUTING` part, idk, I only tested channels V2 and having that key with V2 was crashing (actively rejected, not ignored).